src/pages/index.js

```diff
@@ -116,15 +114,30 @@ function renderCard(cardData) {
             cardDeleteModal.renderLoading(false);
           }); 
       });
-      card.processLikeClick();
+      
     },
-   
-
+    processLikeClick,
     userId,
   });
 
   section.addItem(card.getCard());
 }
+function processLikeClick(card) {
+  const isLiked = card.isLiked();
+  if (isLiked) {
+    () => {
+      this.setLikesInfo(data.likes);
+    }
+    api.removeCardLike(card._cardId).then((data) => {
+      this.setLikesInfo(data.likes);
+    });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**

Reply:
done